### DISCUSSION TITLE
How do services that scale up affect billing?

### DISCUSSION THREAD

**Discussion by dangdennis** [encoredev/encore#275] [Help] [answered]
If my app quickly grows in a few days or weeks with sustained traffic, how does this affect the pricing for the various tiers?

Good thing VC money is here to help /joke 😂

**Comment by dangdennis** [comment]
Okay, basic reasoning. Free tier scaling limited. But since we use our own cloud, obv I pay for additional infra. 

Do you have a way to predict operations cost short of me going into AWS billing docs?

**Reply by marcuskohlberg** [reply, marked as answer]
Hey @dangdennis, great questions.

As you've already identified, Encore won't charge for usage since for production use cases, you'll use your own cloud account with e.g. AWS. When Encore provisions your infrastructure, you can see exactly what that looks like in your AWS account, as you normally would. And all of AWS tools for monitoring and predicting billing work as they normally would.

We currently don't have any feature built-in for predicting costs, but this is definitely something we're planning to add. Along with tools for cost optimization, taking advantage of the fact that Encore actually understands how your application fits together!
This is what's great about Encore not being a hosting provider – there's no incentive to have you spend unnecessarily, which can't be said for the cloud providers.

Hope that helps, if you've more specific questions let me know.
